athena-gcs/src/test/java/com/amazonaws/athena/connectors/gcs/GcsMetadataHandlerTest.java

```diff
@@ -319,5 +319,7 @@ public void testDoGetSplits() throws Exception
         when(table.getPartitionKeys()).thenReturn(columns);
         GetSplitsResponse response = gcsMetadataHandler.doGetSplits(blockAllocator, request);
         assertNotNull(response);
+        assertEquals(2, response.getSplits().size());
+        assertNotNull(response.getSplits().stream().findFirst().get().getProperty("year"));
```

Comment:
Why are you asserting only that its not null and only asserting the first value?